### DISCUSSION TITLE
Heimdell error 500

### DISCUSSION THREAD

**Discussion by gamersalpha** [tteck/Proxmox#2518] [Help Needed] [answered]
Hello there, how debug my heimdell instance, i got error 500 

how to debug please ?
![image](https://github.com/tteck/Proxmox/assets/32466950/449a087b-edbc-4f81-869d-c93b6e9d67f8)

![image](https://github.com/tteck/Proxmox/assets/32466950/60250906-3aad-46ed-b254-281aebe92adc)

![image](https://github.com/tteck/Proxmox/assets/32466950/8ab08fb9-66ac-47e1-8041-ccb1eb208c5c)

![image](https://github.com/tteck/Proxmox/assets/32466950/d78a98e3-76f9-4d3d-b2b8-8d73cf49f9ba)

please how found log ?

how fix this 500 error please
thanks

**Comment by AlexGeek77** [comment]
Did you just update your installation?

**Reply by gamersalpha** [reply]
no i m using it for month and today stop working : 

no i think this is the space disk : 


![image](https://github.com/tteck/Proxmox/assets/32466950/64a76b7e-489f-4f7e-9ef7-631ba9c390c6)


Why he is go full disk ?

**Comment by gamersalpha** [comment]
i found my problem this is this folders : 
![image](https://github.com/tteck/Proxmox/assets/32466950/267dbf3d-0653-4498-8ee7-372b0e3c1347)

and this files : 
![image](https://github.com/tteck/Proxmox/assets/32466950/0c67efdc-259a-4ab1-a935-2f74ed7dfff5)

but files are not readeable : 
![image](https://github.com/tteck/Proxmox/assets/32466950/1a5c1e38-da84-40e6-9fb8-0ddaf102dfae)

why he is so big i can't read it

**Reply by AlexGeek77** [reply, marked as answer]
It's all part of the Systemd Journal Logs. If you don't want to have logs forever (you likely don't need that) then you can restrict the logs to not exceed a certain size with this command
```
journalctl --vacuum-size=100M
```
That should purge 1.0G of logs for you.

**Reply by gamersalpha** [reply]
but this log of my container or my real proxmox serveur ?

**Reply by AlexGeek77** [reply]
It's just for your LXC.
You can read up on other journalctl commands here:
https://linuxhandbook.com/clear-systemd-journal-logs/

**Reply by gamersalpha** [reply]
Works : 

![image](https://github.com/tteck/Proxmox/assets/32466950/b03b32ce-ec5b-4fbd-acba-d8d04e4df09e)


but i realy want understand what is wrong with this logs files :/

i have to install wazuh
